mne/channels/tests/test_montage.py

```diff
@@ -40,68 +40,90 @@
-        'very_very_very_long_name   -60    -51\n'
-        'dummy -60 -52\n',
-        '346\n'
-        'EEG\t      F3\t -62.027\t -50.053\t      85\n'
-        'EEG\t      Fz\t  45.608\t      90\t      85\n'
-        'EEG\t      F4\t   62.01\t  50.103\t      85\n'
-        'EEG\t      FCz\t   68.01\t  58.103\t      85\n',
-        'eeg Fp1 -95.0 -3. -3.\neeg AF7 -1 -1 -3\neeg A3 -2 -2 2\neeg A 0 0 0'
-    ]
-
-    kinds = ['test.sfp', 'test.csd', 'test_mm.elc', 'test_m.elc', 'test.txt',
-             'test.elp', 'test.hpts']
-    for kind, text in zip(kinds, input_str):
-        fname = op.join(tempdir, kind)
+    inputs = dict(
+        sfp='FidNz 0       9.071585155     -2.359754454\n'
+            'FidT9 -6.711765       0.040402876     -3.251600355\n'
+            'very_very_very_long_name -5.831241498 -4.494821698  4.955347697\n'
+            'Cz 0       0       8.899186843',
+        csd='// MatLab   Sphere coordinates [degrees]         Cartesian coordinates\n'  # noqa: E501
+            '// Label       Theta       Phi    Radius         X         Y         Z       off sphere surface\n'  # noqa: E501
+            'E1      37.700     -14.000       1.000    0.7677    0.5934   -0.2419  -0.00000000000000011\n'  # noqa: E501
+            'E3      51.700      11.000       1.000    0.6084    0.7704    0.1908   0.00000000000000000\n'  # noqa: E501
+            'E31      90.000     -11.000       1.000    0.0000    0.9816   -0.1908   0.00000000000000000\n'  # noqa: E501
+            'E61     158.000     -17.200       1.000   -0.8857    0.3579   -0.2957  -0.00000000000000022',  # noqa: E501
+        mm_elc='# ASA electrode file\nReferenceLabel  avg\nUnitPosition    mm\n'  # noqa:E501
+               'NumberPositions=    68\n'
+               'Positions\n'
+               '-86.0761 -19.9897 -47.9860\n'
+               '85.7939 -20.0093 -48.0310\n'
+               '0.0083 86.8110 -39.9830\n'
+               '-86.0761 -24.9897 -67.9860\n'
+               'Labels\nLPA\nRPA\nNz\nDummy\n',
+        m_elc='# ASA electrode file\nReferenceLabel  avg\nUnitPosition    m\n'
+              'NumberPositions=    68\nPositions\n-.0860761 -.0199897 -.0479860\n'  # noqa:E501
+              '.0857939 -.0200093 -.0480310\n.0000083 .00868110 -.0399830\n'
+              '.08 -.02 -.04\n'
+              'Labels\nLPA\nRPA\nNz\nDummy\n',
+        txt='Site  Theta  Phi\n'
+            'Fp1  -92    -72\n'
+            'Fp2   92     72\n'
+            'very_very_very_long_name       -92     72\n'
+            'O2        92    -90\n',
+        elp='346\n'
+            'EEG\t      F3\t -62.027\t -50.053\t      85\n'
+            'EEG\t      Fz\t  45.608\t      90\t      85\n'
+            'EEG\t      F4\t   62.01\t  50.103\t      85\n'
+            'EEG\t      FCz\t   68.01\t  58.103\t      85\n',
+        hpts='eeg Fp1 -95.0 -3. -3.\n'
+             'eeg AF7 -1 -1 -3\n'
+             'eeg A3 -2 -2 2\n'
+             'eeg A 0 0 0',
+    )
```

Comment:
Ha, I thought there was some purpose to it being formatted really ugly ... Looks much better this way.